test/python/transpiler/test_vf2_layout.py

```diff
@@ -145,10 +145,11 @@ def test_2q_circuit_nested_control_flow(self, strict_direction):
 
     def test_3q_circuit_3q_coupling_non_induced(self):
         """A simple example, check for non-induced subgraph
+
             1         qr0 -> qr1 -> qr2
            / \
           0 - 2
-        """
+        """  # noqa: D208
```

Comment:
Can probably disable all docstring-related checks within the tests, since they're not used for anything other than just being free-form comments on the test.

Reply:
Hmm. Not sure I agree 100%. Maybe there are some that should be disabled in tests and others not? I'll have to look at it in more detail.

Reply:
Test docstrings just aren't built into anything.  If the purpose of docstring lints is to try and provide a more consistent/polished docs experience, then they just don't apply to tests - we only even _have_ test docstrings at the moment because the pylint rules are enforcing it somewhat accidentally.  Ideally in most cases, the test name should give enough of a description (since that's what appears in logs), and the docstring/comment can contain a little more information in cases where that actually helps.

Reply:
+1. Test docstrings are annoying and often don't add any new info compared to the test's name

Reply:
I'm not opposed to removing docstrings in test. But I don't like allowing docstrings with no style rules. All code serves as an example of how to code. Having bad examples in the code base is counter productive.